dojo/api_v2/serializers.py

```diff
@@ -693,9 +693,25 @@ def update(self, instance, validated_data):
 
         instance = super(TaggitSerializer, self).update(instance, validated_data)
 
+        # Track if we will be doing an extra save.
+        save_instance = push_to_jira
+
+        # Allow setting or clearing the mitigation date based upon the state of is_Mitigated.
+        if instance.is_Mitigated and instance.mitigated is None:
+            save_instance = True
+            instance.mitigated = datetime.datetime.now()
+            instance.mitigated_by = self.context['request'].user
+            if settings.USE_TZ:
+                instance.mitigated = timezone.make_aware(instance.mitigated, timezone.get_default_timezone())
+        elif not instance.is_Mitigated and instance.mitigation is not None:
```

Comment:
should this check for `instance.mitigated` instead of `instance.mitigation` ?

Reply:
Good catch. My testing didn't catch this. Will fix.